src/_includes/feature_lists/cloud.njk

```diff
@@ -82,6 +87,11 @@
             label: "Seamless Data Stream",
             values: [null, 'check', 'check'],
             info: "<p><a href=\"https://github.com/FlowFuse/nr-project-nodes\" target=\"blank\">FlowFuse Project Nodes</a> enable the passing of data and messages between your Node-RED projects, included with all FlowFuse Projects by default.</p>"
+        }, {
+            id: "certified-nodes",
+            label: "Certified Nodes",
+            values: [null, 'check', 'check'],
+            info: "<p>FlowFuse Certified Nodes a selection of certfied Node-RED nodes, enhancing efficiency and security.</p>"
```

Comment:
This should get a page under `/product/` and link to the documentation.

Reply:
Yes, I would add the link later. Also in preparation is a blog article about Certified Nodes